utils.go

```diff
@@ -7,13 +7,12 @@ import (
 	"log"
 	"math"
 	"os"
-	"runtime"
 	"runtime/pprof"
 	"strings"
 	"time"
 
 	"github.com/google/uuid"
-	"github.com/shirou/gopsutil/process"
+	"github.com/prometheus/procfs"
```

Comment:
The new dependency should be aslo added to .travis.yml, or it will break our testing.